src/pages/Profile.js

```diff
@@ -1,6 +1,10 @@
 import React, { useEffect, useState, useContext } from 'react';
 import PropTypes from 'prop-types';
-import { ChartContainer } from '@codeforafrica/hurumap-ui';
+import {
+  ChartContainer,
+  EmbedPopup,
+  InfoPopup
```

Comment:
I'd call these both "DropDowns" and change "Embed" to just "Share"? cc @kilemensi

Reply:
@DavidLemayian Clarity on this?